tests/components/bayesian/test_binary_sensor.py

```diff
@@ -76,7 +77,7 @@ def test_unknown_state_does_not_influence_probability(self):
         assert setup_component(self.hass, "binary_sensor", config)
 
         state = self.hass.states.get("binary_sensor.test_binary")
-        assert state.attributes.get("observations") == [None]
+        assert state.attributes.get("observations") == []
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)